docker-imposm3/Dockerfile

```diff
@@ -1,7 +1,16 @@
+FROM golang:1.16.0-buster as mother
+MAINTAINER Asdrubal Gonzalez <asdrubal.gonzalez@geo-solutions.it>
+
+WORKDIR /code
+#Install the tool for merging the PBF files.
+RUN apt update && apt install osmium-tool -yq
+COPY ./*.pbf /code/
+RUN osmium merge denmark-latest.osm.pbf finland-latest.osm.pbf iceland-latest.osm.pbf norway-latest.osm.pbf sweden-latest.osm.pbf -o /code/scandinavia-latest.osm.pbf
```

Comment:
Instead of having multiple dockerfiles for each combination of PBF to import, why don't you make it more generic?

Something along these lines:
COPY ./*.pbf /code/
RUN osmium merge *.pbf -o /code/merged.pbf